guides/theming.md

```diff
@@ -1,311 +1,384 @@
-# Theming your Angular Material app
+# Theming Angular Material
 
-## What is a theme?
+## What is theming?
 
-Angular Material's theming system enables you to customize components to
-better reflect your product's brand. A theme consists of configurations for the individual
-`color` and `typography` systems in Angular Material. The library's approach to theming reflects
-the guidance from the [Material Design spec][1].
+Angular Material's theming system lets you customize color and typography styles for components
+in your application. The theming system is based on Google's
+[Material Design][material-design-theming] specification.
 
-In Angular Material, you create a color configuration by composing multiple palettes. In
-particular, a color configuration consists of:
+This document describes the concepts and APIs for customizing colors. For typography customization,
+see [Angular Material Typograpy][mat-typography]. For guidance on building components to be
```

Comment:
typo: `Typograpy`

Reply:
Re-opening - still says `Typograpy` in `[Angular Material Typograpy]`

Reply:
fixed for real now